common/api/imodeljs-common.api.md

```diff
@@ -3971,11 +3944,11 @@ export interface IModelVersionProps {
 
 // @internal
 export abstract class IModelWriteRpcInterface extends RpcInterface {
-    // (undocumented)
+    // @deprecated (undocumented)

```

Comment:
What should an app use instead of deprecated `IModelWriteRpcInterface` methods?

Reply:
> In display-test-app, the following key-ins work:
> 
> * `dta edit` to start/end interactive editing session
> * `dta delete elements`
> * `dta move elements`
> * `dta place line string`
>   The following have no effect any more:
> * `dta undo`
> * `dta redo`

should be fixed now (but I didn't test)

Reply:
all deprecated methods say: BriefcaseConnection with IpcHost/IpcApp